trtools/utils/tr_harmonizer.py

```diff
@@ -680,6 +682,15 @@ def __init__(self,
             self.has_fabricated_ref_allele = False
             self.ref_allele_length = len(ref_allele) / len(motif)
 
+        # declaration of end_pos variables
+        self.end_pos = int(self.pos + self.ref_allele_length * len(motif) - 1)
+        self.full_alleles_end_pos = self.end_pos
```

Comment:
A few comments:
* I would rather not define this here and then redefine it later if we have full_alleles, that could lead to confusion for a reader. Let's just have "if full_alleles is None then .... else ...."
* If we're going to define this, can we document this (in the same way you documented end_pos). It can go in the attributes section. 
* It seems `full_alleles_pos` is already defined but not documented, do you mind documenting that likewise?

Reply:
I've applied your suggestions.